storage/reads/array_cursor.gen.go.tmpldata

```diff
@@ -43,6 +43,14 @@
 				"Accumulate":"sum += a.Values[rowIdx]; count++",
```

Comment:
Is there any reason to keep the plain "Mean" aggregations?

Reply:
I think yes. We currently still use plain `Mean` for OSS, and could potentially optimise enterprise to use the plain `Mean` in the fully replicated case.